agent/bench-scripts/pbench-fio

```diff
@@ -679,47 +677,62 @@ function fio_run_job() {
 	pbench-send-tools --group=$tool_group --dir=$benchmark_results_dir
 	pbench-postprocess-tools --group=$tool_group --dir=$benchmark_results_dir
 
-	if [ ! -z "$clients" ]; then
-		histogram_params="--time-quantum ${histogram_interval_sec} --percentiles 0 50 90 95 99 100"
-		histogram_cmd="fio-histo-log-pctiles.py ${histogram_params} -- "
-		for client in `echo $clients | sed -e s/,/" "/g`; do
-			mkdir -p $benchmark_results_dir/clients/$client
-			/bin/mv $benchmark_results_dir/*.log.$client $benchmark_results_dir/clients/$client/ 2> /dev/null
+	local histogram_params="--time-quantum ${histogram_interval_sec} --percentiles 0 50 90 95 99 99.5 100"
+	local histogram_cmd="fio-histo-log-pctiles.py ${histogram_params} -- "
+
+	# First move all fio log files to the appropriate client directory.
+	if [ ! -z "${clients}" ]; then
+		local client
+		for client in `echo ${clients} | sed -e s/,/" "/g`; do
+			mkdir -p ${benchmark_results_dir}/clients/${client}
+			/bin/mv ${benchmark_results_dir}/*.log.${client} ${benchmark_results_dir}/clients/${client}/ 2> /dev/null
```

Comment:
Why are we using the full path to `mv` here and below? Guessing unit-test related: mocking `mv` does not affect these?

Reply:
I am guessing, but not sure entirely.  I tried not to change too much here.